CHANGELOG.md

```diff
@@ -1,5 +1,46 @@
 # Changelog
 
+## v0.34.7
+
+*February 18, 2021*
+
+This release fixes a downstream security issue which impacts Cosmos SDK
+users who are:
+
+* Using Cosmos SDK v0.40.0 or later, AND
```

Comment:
Do we want to add something like v0.40.0 **to v0.41.3** or whatever it was when this issue was fixed

Reply:
There weren't any changes on the SDK side to fix this